phoenix-core-client/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -1240,6 +1241,15 @@ private void send(Iterator<TableRef> tableRefIterator) throws SQLException {
                     String tableName = tableRef.getTable().getTableName().toString();
                     mc.updateCache(tenantId, schemaName, tableName, true);
                 }
+            } catch (MetaDataEntityNotFoundException e) {
```

Comment:
Why would `ValidateLastDDLTimestampUtil#validateLastDDLTimestamp` throw `MetaDataEntityNotFoundException` ?

Reply:
Removed that change.